apps/myaccount/src/utils/profile-utils.ts

```diff
@@ -30,6 +30,7 @@ import {
 import { store } from "../store";
 import { setProfileCompletion } from "../store/actions";
 import { SCIMConfigs } from "../extensions/configs/scim";
+import { commonConfig } from "../extensions";
```

Comment:
```suggestion
import { SCIMConfigs } from "../extensions/configs/scim";
import { commonConfig } from "../extensions";
```
I think we can combine these two imports.